### DISCUSSION TITLE
MaskInput/OTPInput/PhoneNumberInput

### DISCUSSION THREAD

**Discussion by darthwade** [VKCOM/VKUI#2748] [Ideas] [answered]
Hey guys,

I think it's good idea to add such components to the lib, as those are essential to any project nowadays :)

Thanks for your great job, lib is awesome ;)

**Comment by stoope** [comment, marked as answer]
@darthwade Hi! Thanks for the good suggestion! We've done some research on this topic and found out that `mask` implementation adds about `15kb` of minified code to the bundle, so we can't afford it until we implement code splitting. However, you can use any `inputMask` library such as [react-number-format](https://www.npmjs.com/package/react-number-format). it works smoothly with `VKUI` `Input`.
`<NumberFormat customInput={Input} format="#### #### #### ####" />`

**Reply by darthwade** [reply]
Got it, thanks @stoope  ;)
